airflow/models/__init__.py

```diff
@@ -2563,6 +2568,25 @@ def pre_execute(self, context):
         """
         pass
 
+    def get_executor_config(self, executor_config):
+        """Try to import base_executor_config and merge supplied
+        executor_config into it.
+
+        :return: dict"""
```

Comment:
not sure about this PyCharm return.

Reply:
Oh that the doc string is not below? Let me look into that.